libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
+  color?: string | undefined;
+  backgroundColor?: string | undefined;
+  borderRadius?: number | undefined;
+  onPress?: void;
   name: IconType;
+  iconStyle?: object | undefined;
```

Comment:
Don't use object as type, could we have some interface instead?

Reply:
Interface sounds good, but dunno what do you mean by that. What do expect in interface then ?

Reply:
I agree with @pmusialik-intive